### DISCUSSION TITLE
Hackathons related to Medicine

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [EddieHubCommunity/support#885] [Tips and Tricks]
Insilicobiology and I had a conversation about Hackathons related to the medical field on discord recently. If you'd like to take a look at it, please check it out here: [Link to a Discord message](https://discord.com/channels/699608417039286293/699609751914414161/807321695769329684)

I'll be copy-pasting the conversation we had over here. If you haven't joined our server yet, you can do so here: [Server Invite](https://discord.gg/jZQs6Wu)

**Comment by Akshu-on-github** [comment]
A brief transcript of our conversation:

**Akshu-on-github:**
>Does your institute have any datasets that are open to the public?
>I have a Hackathon related to the medical field next week, so I'm on the lookout for some good ones

**insilicobiology:**
>Look for Gene expression omnibus. It's the biggest genomics open repository. The research facility where I work exists within a research hospital so we don't publish patient data
>[Link to Gene Expression Omnibus](https://www.ncbi.nlm.nih.gov/geo/)
> You can research the database for cell types, specific illnesses and specific protocols of stimulation for cell types, as well as species :smile:

**Akshu-on-github:**
> Ah, I understand- patient confidentiality, right?

**insilicobiology:**
> Absolutely.  There's a ton of problems that can happen to patients if their medical data gets revealed. Imagine if, for example, an AIDS patient gets his/her status as HIV+ revealed. There's a lot of pushback and discrimination associated with some diseases and researchers are the guardians of the data that the patients kindly gave us to advance medicine. respect goes both ways and we need to respect the trust the patients gave us.

> My 4 pieces of advice to the people in the hackathon.
> 1. Non-techies in medicine usually have little time and have but a basic grasp of statistics. Work on keeping everything simple and easily explainable
> 1. Neural networks are not the answer to everything, especially when they are so hard to get the reasoning to an answer from them. Medicine without explainability is not medicine.
> 1. biology datasets are full of noise and with little nuggets of gold. a gain of 5% to 10% is a win in most medicine computational biology
> 1. Keep a reasonable and standardized set of variables and always register their meaning. it's incredibly easy to mistake similar gene naming conventions for example :smile:

**Akshu-on-github:**
>Thanks a million... I really appreciate it - I probably would have gone around in circles if I hadn't read this
But I do understand - medical conditions vary in a case by case basis, right? Pre-existing conditions and medical history play a huge role in it

**insilicobiology:**
> Yep, there has been quite a boom on what's called translational and personalized medicine which means fitting the treatment to the patient's genetic background and their medical history. It's a lot of data sources in this line of research but allows managing difficult diseases with more efficacy and provide just the right amount of treatment. BTW if you get stuck with gene naming conventions search for biomart+gene naming convention. Biomart is the most reliable way to translate gene names from one naming convention to another
[Link to BioMart help](https://www.genenames.org/help/biomart/)

Thanks a bunch, @insilicobiology - I learnt a lot from our short chat and I'm sure that others reading this will too!

**Reply by eddiejaoude** [reply]
Thanks for sharing 👍
